### PR TITLE
Write basic PKGBUILD

### PR DESCRIPTION
**Pull request by marzipankaiser**
Add a basic PKGBUILD for archlinux

@mee4895 @Vogel612 Feedback?

@mee4895 Maybe the dmenu script could be packaged, too. Opinions? (maybe via optdepends dmenu)

### CODE REVIEW

File: PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+pkgdesc="A CLI for the FIUS drinklist"
+arch=('any')
+url="https://github.com/FIUS/drinklist-cli"
+license=('GPL')
+depends=(python python-requests python-numpy python-appdirs)
+source=("https://github.com/FIUS/drinklist-cli/archive/master.tar.gz")
+sha256sums=('cb5bfac7835ea157c3ad07712fca5460075fd3ad47346d16cdbcc1856ca8aedb')
+
+build() {
+	cd "$srcdir/$pkgname-$pkgver"
```
Comment (Vogel612) [inline PKGBUILD]: indentation mismatch?

File: PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    ./build.sh
+}
+
+package() {
+	cd "$srcdir/$pkgname-$pkgver"
```
Comment (Vogel612) [inline PKGBUILD]: indentation mismatch

**Comment by Vogel612** [comment]
Not tested yet, but seems good.

File: PKGBUILD (hunk 2 above)
Comment (marzipankaiser) [reply PKGBUILD]: Should be fixed now.

File: PKGBUILD (hunk 1 above)
Comment (marzipankaiser) [reply PKGBUILD]: Should be fixed now.

**Comment by Vogel612** [comment]
Building and installing appears to work correctly.

**Comment by marzipankaiser** [comment]
Thanks.

**Pull request merged**